frontend/packages/kubevirt-plugin/src/components/vms/vm-details-page.tsx

```diff
@@ -90,11 +90,11 @@ export const VirtualMachinesDetailsPage: React.FC<VirtualMachinesDetailsPageProp
 
   const resources = [
     getResource(VirtualMachineInstanceModel, {
```

Comment:
Looks fine to me.
But for another developers it would seem odd to use list and not a single object.  
Thus, I think it's best to add a comment somewhere to explain why we are using list (IMO here it's a good place).
maybe even add the BZ url.

Reply:
thanks, done :+1: